src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -59,6 +59,7 @@
   private static final String DEV_APPSERVER_PY = "bin/dev_appserver.py";
   private static final String JAVA_APPENGINE_SDK_PATH =
       "platform/google_appengine/google/appengine/tools/java/lib";
+  private static final String JAVA_COMPONENTS_PATH = "platform/gcd/.appengine";
```

Comment:
I don't think we want anything depending on these internal paths as they are not necessarily stable.  I don't even think JAVA_APPENGINE_SDK_PATH is appropriate.  If you want to check components, you should probably shell out and use --format to get the output back in a machine readable format.

What exactly are you trying to detect the presence of?  The gcd directory is for the datastore emulator which is currently deprecated in favor of the new version.  Is this what you are looking for?